### PR TITLE
Improvements to server 4 VM service docs

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
Docs for VM service in server 4 need improvements, and there are some missing sections:

- [x] Add in Arm AMI list
- [x] Add in info for WIndows images for AWS and GCP
- [x] Add in info about preallocated instances
- [x] Add in infor about alternative linux images
- [x] Move all installation info into the installation guide. The VM service page in the operators guide should just be for config options that can be applied post installation.

# Reasons
https://circleci.atlassian.net/browse/DOCTEAM-779?atlOrigin=eyJpIjoiMjI4YjU2YzZjMTdiNDY5MmI3ZWExYmY3ZDlmYjM5ZjQiLCJwIjoiaiJ9

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/server/operator/manage-virtual-machines-with-vm-service.adoc [hunk 1]
```diff
@@ -128,39 +123,144 @@ The default AMIs for server v4.x are based on Ubuntu 22.04.
...
+
+[#apply-changes]
+== Apply changes
+
+Apply the changes made to your `values.yaml` file:
+
+[source,shell,subs=attributes+]
+----
+namespace=<your-namespace>
+helm upgrade circleci-server oci://cciserver.azurecr.io/circleci-server -n $namespace --version {serverversion4} -f <path-to-values.yaml>
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/operator/manage-virtual-machines-with-vm-service.adoc:265]: Pulling in current server version here so always up to date, I think thats what we want?

File: jekyll/_cci2/server/operator/manage-virtual-machines-with-vm-service.adoc [hunk 2]
```diff
@@ -128,39 +123,144 @@ The default AMIs for server v4.x are based on Ubuntu 22.04.
...
-
-      service_account: <service-account-json>
-      # OR
-      workloadIdentity: ""  # Leave blank if using JSON keys of service account else service account email address
+      ...
+      linuxImage: "<my-linux-image>"
+----
+
+[#instance-preallocation]
+== Instance preallocation
```
[the remark this answers is not in the document]
Comment (lloydChris) [reply jekyll/_cci2/server/operator/manage-virtual-machines-with-vm-service.adoc:219]: I prefer "preallocated", or even "on standby", although i see we use the term pre-scale in the values.yaml, so maybe that is better for consistency 🤷

**Review by lloydChris — APPROVED** [review]
a nit.  Otherwise looks good

**Review by ryanwohara — APPROVED** [review]

File: jekyll/_cci2/server/operator/manage-virtual-machines-with-vm-service.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (rosieyohannan) [reply jekyll/_cci2/server/operator/manage-virtual-machines-with-vm-service.adoc:219]: I'm just going to leave it for now, all other server version's docs say preallocate. Can always revisit this.

**Review by devinmarieb — APPROVED** [review]

**Pull request merged**